Documentation/ApiOverview/Events/Events/Backend/_AfterRawPageRowPreparedEvent/_MyEventListener.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyVendor\MyExtension\Backend\EventListener;
+
+use TYPO3\CMS\Backend\Tree\Repository\AfterRawPageRowPreparedEvent;
+use TYPO3\CMS\Core\Attribute\AsEventListener;
+
+final class MyEventListener
+{
+    #[AsEventListener]
+    public function __invoke(AfterRawPageRowPreparedEvent $event): void
+    {
+        $rawPage = $event->getRawPage();
+        if ((int)$rawPage['uid'] === 123) {
+            // Sort pages alphabetically in the page tree
+            $rawPage['_children'] = usort(
+                $rawPage['__children'],
```

Comment:
Should be one `_`, right?

```suggestion
                $rawPage['_children'],
```

Reply:
Yes, it's ONE underscore. Just checked in TYPO3 sources.